tket/src/Transformations/Rebase.cpp

```diff
@@ -59,7 +59,7 @@ static bool standard_rebase(
         tk1_replacement) {
   bool success = false;
   VertexSet bin;
-  BGL_FORALL_VERTICES(v, circ.dag, DAG) {
+  for (const Vertex& v : circ.vertices_in_order()) {
```

Comment:
Why do we need these changes?

Reply:
Without this change, a rebase pass operating on a circuit with a conditional operation enters an infinite loop, as the vertices in the DAG are modified inside the iteration. I don't quite get why we didn't see this before, maybe just luck...?

Reply:
That's interesting... anyway, the code looks safer now. However, do we really need a sorted list?

Reply:
Done.